names_mapping

```diff
@@ -0,0 +1,63 @@
+string,map to
+(including Ilex verticillata),Ilex verticillata
+(including introduced Ulmus species),Ulmus
+(including Populus deltoides),Populus deltoides
+Acer (including Acer negundo),Acer negundo
+all Malus (including Malus domestica),Malus domestica
+all Prunus,Prunus
+aphids,DISCARD
+Betula (including Betula alleghaniensis),including Betula alleghaniensis
+Betula (including Betula papyrifera),including Betula papyrifera
+Caloptilia syringella (Fabricius)**- Exotic,Caloptilia syringella (Fabricius)
+Carya (including Carya cordiformis,Carya cordiformis
+Carya (including Carya cordiformis),Carya cordiformis
+Carya (including Carya illinoinensis,Carya illinoinensis
+Carya (including Carya illinoinensis),Carya illinoinensis
+Carya ovata),Carya ovata
+Coniferous Species,Gymnosperms
+Decaying Organic Matter,DISCARD
+Diospyros (including Diospyros virginiana),Diospyros virginiana
+Gleditsia or Robinia (locust),Fabaceae
+Juglans (including Juglans cinerea),Juglans cinerea
+Laurus nobilis or Magnolia virginiana (sweetbay),DISCARD
+Lindera (including Lindera benzoin),Lindera benzoin
+Malus (including Malus domestica),Malus domestica
+Mesquite,Prosopis
+mites),DISCARD
+mites),DISCARD
+Morus (including Morus rubra),Morus rubra
+Nut Trees,DISCARD
+Other Acer Species,Acer
+Other Species in Olive Family,Burseraceae
+Plantain,Plantago
+Populus (including Populus deltoides),Populus deltoides
+Populus (including Populus trichocarpa),Populus trichocarpa
+Predatory on Insects,DISCARD
+Predatory on Insects (scales,DISCARD
+Prunus (including Prunus armeniaca,Prunus armeniaca
+Prunus (including Prunus armeniaca),Prunus armeniaca
+Prunus (including Prunus avium),Prunus avium
+Prunus (including Prunus cerasifera,Prunus cerasifera
+Prunus (including Prunus persica,Prunus persica
+Prunus (including Prunus persica),Prunus persica
+Prunus (including Prunus virginiana),Prunus virginiana
+Prunus pensylvanica),Prunus pensylvanica
+Prunus persica variety nectarina),Prunus persica var. nectarina
+Prunus persica),Prunus persica
```

Comment:
Is this to chop of the straggling ```)``` ?

Reply:
yes- if that'll be handled downstream, then it's not needed